connection_pool/example_test.go

```diff
@@ -11,7 +11,7 @@ import (
 type Tuple struct {
 	// Instruct msgpack to pack this struct as array, so no custom packer
 	// is needed.
-	_msgpack struct{} `msgpack:",asArray"` //nolint: structcheck,unused
```

Comment:
AFAIS, those comments don't land to the generated documentation, so we can keep `//nolint: structcheck,unused` here (and in all othre places) and remove them from the global flags.

Reply:
I have found this one: 
<img width="625" alt="image" src="https://user-images.githubusercontent.com/55545103/165343509-05b3e40b-4195-4cae-9c62-193b6704cbfc.png">

Reply:
I think it is okay in the example. A user will likely copy-paste it, at least partially, and if something that requires a linter exception is used, it is good to mark it just like a user will do in the resulting code.

Reply:
Okay, I leave it so.